HACKING.md

```diff
@@ -122,6 +122,18 @@ git submodules:
 Refer to the [testing README](test/README.md) for details on running the Cockpit
 integration tests locally.
 
+## Testing the Python bridge
+
+There is currently an experimental replacement for `cockpit-bridge` being
+written in Python.  It lives in `src/cockpit` with most of its rules in
+`src/Makefile.am`.  This directory was chosen because it matches the standard
+so-called "src layout" convention for Python packages, where each package
+(`cockpit`) is a subdirectory of the `src` directory.
+
+You can run a subset of the unit tests against this experimental code by typing
+`make pycheck`.  Note that this needs to be done against a git checkout, as the
+Python files are not currently included in tarball releases.
```

Comment:
Do we want to shortly mention that you have to `make cockpit-bridge.pyz` to build it and for example can test it with cockpit-client? After dropping `cockpit-bridge` in `~/.local/bin/`

Reply:
Sure.

Reply:
Done.